Src/Persistence/Configurations/LogConfiguration.cs

```diff
@@ -0,0 +1,21 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+using TeamProject.Domain.Entities;
+
+namespace TeamProject.Persistence.Configurations
+{
+    public class LogConfiguration : IEntityTypeConfiguration<Log>
+    {
+        public void Configure(EntityTypeBuilder<Log> builder)
+        {
+            builder.HasKey(e => e.LogId);
+            builder.Property(e => e.Application).IsRequired().HasMaxLength(64);
+            builder.Property(e => e.Time).IsRequired();
+            builder.Property(e => e.Level).IsRequired().HasMaxLength(64);
+            builder.Property(e => e.Message).IsRequired().HasMaxLength(512);
+            builder.Property(e => e.Service).IsRequired(false).HasMaxLength(512);
+            builder.Property(e => e.Exception).IsRequired(false).HasMaxLength(512);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Assign this magic number '512' to a well-named (variable|constant), and use the (variable|constant) instead.](https://app.codacy.com/manual/liannoi/team-project/pullRequest?prid=5600534)